src/_shared/scss/_adverts-masterclasses.scss

```diff
@@ -1,42 +1,48 @@
+
+.adverts--tone-masterclasss .button--large {
+    background: #ffffff;
+    color: #000000;
+}
+
 .adverts--masterclasss {
 }
 
 .adverts--tone-masterclasss {
     > .adverts__header {
-        background: $multimedia-main-1;
+        background: $guardian-masterclasses-rebrand;
         .button {
-            color: $multimedia-main-2;
-            border-color: fade-out($multimedia-main-2, .7);
+            color: #000000;
```

Comment:
Is this required? Buttons are already styled as black above (with `.button-large`). Try removing it and see.

Reply:
Absolutely right, it's not doing anything - now removed.